.gitignore

```diff
@@ -1,3 +1,3 @@
-mpact_venv/
+*_venv/
```

Comment:
any reason for this, other than having other envs in your own set up?

Reply:
Yeah, that is the reason. I can change it back if you prefer.

Reply:
No, it is fine, this was just for my own understanding